velox/common/base/PrefixSortConfig.h

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#pragma once
+
+#include <stdint.h>
+
+namespace facebook::velox::common {
+
+/// Specifies the config for prefix-sort.
+struct PrefixSortConfig {
+  PrefixSortConfig(int64_t maxNormalizedKeySize, int32_t threshold = 130)
```

Comment:
linter error: add 'explicit' specifier. Reason: A single-argument constructor may inadvertently be used as a type conversion constructor. Prefix the constructor with the 'explicit' keyword to avoid this.